R/annotatePeakInBatch.R

```diff
@@ -1,37 +1,40 @@
-#' Obtain the distance to the nearest TSS, miRNA, and/or exon for a list of
-#' peaks
+#' Obtain the distance to the nearest TSS, miRNA, and/or exon for a list of peaks
```

Comment:
the newline here is used to follow 80 characters per line. Do not need to change.